source/hardware/controllers/ion_discover_dj_pro.rst

```diff
@@ -0,0 +1,88 @@
+ION Discover DJ Pro
+===================
+
+Ion Discover DJ Pro is a 2-deck controller with integrated audio interface and iPhone dock.
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro (schematic view)
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro (schematic view)
+
+-  `Mixxx User Forum <https://mixxx.discourse.group/t/ion-discover-dj-pro-mapping-script/12120>`__
+
+.. versionadded:: 2.2.5
```

Comment:
Oops, there will be no 2.2.5, I'll change this to 2.3.

Reply:
Ok, thank @Holzhaus 

But.. what about the GitHub wiki?
Is that no longer maintained?
- I changed the version number also in the table on https://github.com/mixxxdj/mixxx/wiki/Hardware-Compatibility
- What about the https://github.com/mixxxdj/mixxx/wiki/Ion-Discover-DJ-Pro page?